### PR TITLE
wchat Updated to change target person (#217)

### PR DESCRIPTION
**Pull request by zulkernine**
Both io.py and wchat.py are changed to bring the feature of changing target person. User should type '...' alone to change the target person. Two screenshot is attached demonstrating this.
issue217_demonstration1
issue217_demonstration2

### CODE REVIEW

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/utils/io.py  1
- wplay/wchat.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5200199&bid=17054381)

**Comment by zulkernine** [comment]
> Codacy Here is an overview of what got changed by this pull request:
> 
> ```diff
> Complexity increasing per file
> ==============================
> - wplay/utils/io.py  1
> - wplay/wchat.py  2
>          
> ```
> 
> See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5200199&bid=17054381)

What's to do now? @rpotter12 ??

**Comment by rpotter12** [comment]
> > Codacy Here is an overview of what got changed by this pull request:
> > ```diff
> > Complexity increasing per file
> > ==============================
> > - wplay/utils/io.py  1
> > - wplay/wchat.py  2
> >          
> > ```
> > 
> > 
> > See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5200199&bid=17054381)
> 
> What's to do now? @rpotter12 ??

ignore it for now

**Comment by zulkernine** [comment]
Ok

**Comment by xandao6** [comment]
I believe that this PR needs some changes. To start it will only be used for some scripts like wchat and wtrack, right? So you don't need to print : "('...' alone to change target person)" every time.

**Comment by xandao6** [comment]
If you are trying to solve issue # 145 you should also do it in a different PR, so please take the photo of that PR.

File: wplay/wchat.py [hunk 1]
```diff
@@ -13,4 +13,11 @@
 
     while True:
         message = io.ask_user_for_message_breakline_mode()
+        if message[0] == '...':
```
Comment (xandao6) [inline wplay/wchat.py]: And here, don't verify just message[0], will not work every time. Change for:
`if '...' in message:` istead of `if message[0] == '...':`

**Comment by xandao6** [comment]
I suggest you to do a rebase first to solve the conflict!

File: wplay/wchat.py (hunk 1 above)
Comment (zulkernine) [reply wplay/wchat.py]: Ok , I'll update the change soon .

**Comment by zulkernine** [comment]
> If you are trying to solve issue # 145 you should also do it in a different PR, so please take the photo of that PR.

That PR is merged, isn't it?
It showed to be merged, therefore I made this new PR for #217 .

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/utils/io.py  1
- wplay/wchat.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5200199&bid=17054381)

File: wplay/wchat.py [hunk 2]
```diff
@@ -23,7 +23,16 @@
             await target_search.search_and_select_target_without_new_chat_button(page, target)
     else:
         await target_select.manual_select_target(page)
-
+    print("\033[91m {}\033[00m".format("\nType '...' alone in the message to change target person.\n\n"))
     while True:
         message : list[str] = io.ask_user_for_message_breakline_mode()
+
```
Comment (rpotter12) [inline wplay/wchat.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5200199)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5200199&bid=17054381)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5200199&bid=17054381)

File: wplay/wchat.py [hunk 3]
```diff
@@ -24,6 +24,19 @@
     else:
         await target_select.manual_select_target(page)
 
+    print("\033[91m {}\033[00m".format("\nType '...' alone in the message to change target person.\n\n"))
+
     while True:
         message : list[str] = io.ask_user_for_message_breakline_mode()
+
```
Comment (rpotter12) [inline wplay/wchat.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5200199)

Comment (zulkernine) [reply wplay/wchat.py]: ok. I'm removing it in next commit. Please review it now .

**Comment by zulkernine** [comment]
@xandao6  Please review the changes now. 
Screenshot from 2020-03-22 11-38-03
Screenshot from 2020-03-22 11-38-13

File: wplay/wchat.py (hunk 1 above)
Comment (zulkernine) [reply wplay/wchat.py]: I updated the code. So please review it now. I add a screenshot showing the changes. I used formatted string to high light the message how to change target person :)

**Comment by zulkernine** [comment]
I made some changes, so that user can type '...' to send the last message and change the target person simultaneously.

**Comment by xandao6** [comment]
Okay for now

**Pull request merged**